IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -114,7 +115,13 @@ - (void)acquireTokenWithRequest:(MSIDSilentTokenRequest *)request
     CONDITIONAL_START_EVENT(CONDITIONAL_SHARED_INSTANCE, self.requestParameters.telemetryRequestId, MSID_TELEMETRY_EVENT_API_EVENT);
     self.currentRequest = request;
     [request executeRequestWithCompletion:^(MSIDTokenResult *result, NSError *error)
-    {
+     {
+        MSIDAADRequestErrorHandler *handler = [[MSIDAADRequestErrorHandler alloc] init];
+        if (error && [handler shouldRetryNetworkingFailure:error.code]) {
```

Comment:
Not urgent for this PR, can we make `shouldRetryNetworkingFailure` as a class method?

Reply:
yes, updated to be a class method